l2.py

```diff
@@ -0,0 +1,234 @@
+"""
+This module defines functions related to spinning a devnet op-geth node.
+"""
+
+import os
+import shutil
+import subprocess
+import sys
+
+import libroll as lib
+from paths import OPPaths
+from processes import PROCESS_MGR
+
+
```

Comment:
I would drop the "devnet"  part of the names everywhere that is not strictly "devnet" (i.e. a local developer deployment) related. So any logic we can reuse for production deployment (which ideally should be most of it) shouldn't say "devnet".